bpf/include/bpf/ctx/xdp.h

```diff
@@ -156,28 +156,15 @@ l3_csum_replace(const struct xdp_md *ctx, __u64 off, const __u32 from,
 		__u32 flags)
 {
 	__u32 size = flags & BPF_F_HDR_FIELD_MASK;
-	__sum16 *sum;
-	int ret;
+	__sum16 sum[1];
```

Comment:
why not just `__sum16 sum;` and then pass `&sum` to the different functions?

Reply:
Inside xdp_load_bytes, there is a part that calls
there is a part that calls memcpy.

This is fine if it's an inline asm, but
xdp_load_bytes, it requires memory.

What do you think?

Reply:
Oh, that was my mistake, I apologize. I'll accept the offer right away